src/vmm/src/devices/virtio/iov_deque.rs

```diff
@@ -153,16 +147,21 @@ impl<const L: u16> IovDeque<L> {
 
     /// Create a new [`IovDeque`] that can hold memory described by a single VirtIO queue.
     pub fn new() -> Result<Self, IovDequeError> {
-        let memfd = Self::create_memfd()?;
+        let host_page_size = host_page_size();
+        let bytes = L as usize * std::mem::size_of::<iovec>();
+        let num_host_pages = bytes.div_ceil(host_page_size);
+        let pages_bytes = num_host_pages * host_page_size;
```

Comment:
nit: this computation could be moved into a private function, so we dont duplicate it in `drop`

Reply:
done